### ISSUE TITLE
Can't connect to the cloud mongodb.

### ISSUE THREAD

**Issue by prettydev** [iaincollins/next-auth-example#4]
I need to use the cloud mongodb. but can't connect.
Database url is mongodb+srv://username:password@cluster0-jiigr.mongodb.net/test
```
name: 'MissingDriverError',
    message: 'Wrong driver: "mongodb+srv" given. Supported drivers are: "cordova", "expo", "mariadb", "mongodb", "mssql", "mysql", "oracle", "postgres", "sqlite", "sqljs", "react-native", "aurora-data-api", "aurora-data-api-pg".'
```

**Comment by ndom91** [comment]
Try without '+srv' at the end of your protocol, like this:

`mongodb://username:password@cluster0-jiigr.mongodb.net/test`

**Comment by iaincollins** [comment]
Thanks for the bug report! Using `mongodb+srv://` is now supported!

**Comment by prettydev** [comment]
Sorry, still problems.
```
[next-auth][error][ADAPTER_CONNECTION_ERROR] [
  MongoServerSelectionError: getaddrinfo ENOTFOUND cluster0-jiigr.mongodb.net
      at Timeout._onTimeout (D:\auction\work\social-link\node_modules\mongodb\lib\core\sdam\topology.js:430:30)
      at listOnTimeout (internal/timers.js:531:17)
      at processTimers (internal/timers.js:475:7) {
    name: 'MongoServerSelectionError',
    reason: TopologyDescription {
      type: 'Single',
      setName: null,
      maxSetVersion: null,
      maxElectionId: null,
      servers: [Map],
      stale: false,
      compatible: true,
      compatibilityError: null,
      logicalSessionTimeoutMinutes: null,
      heartbeatFrequencyMS: 10000,
      localThresholdMS: 15,
      commonWireVersion: null
    }
  }
]
https://next-auth.js.org/errors#adapter_connection_error
[next-auth][error][GET_USER_BY_PROVIDER_ACCOUNT_ID_ERROR] [
  TypeError: Cannot read property 'getRepository' of null
      at D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:203:44
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:28:103)
      at _next (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:30:194)
      at D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:30:364
      at new Promise (<anonymous>)
      at D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:30:97
      at _getUserByProviderAccountId (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:221:44)
      at getUserByProviderAccountId (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:195:44)
      at D:\auction\work\social-link\node_modules\next-auth\dist\server\lib\callback-handler.js:120:45
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (D:\auction\work\social-link\node_modules\next-auth\dist\server\lib\callback-handler.js:20:103)
      at _next (D:\auction\work\social-link\node_modules\next-auth\dist\server\lib\callback-handler.js:22:194)
]
https://next-auth.js.org/errors#get_user_by_provider_account_id_error
[next-auth][error][OAUTH_CALLBACK_HANDLER_ERROR] [
  Error: GET_USER_BY_PROVIDER_ACCOUNT_ID_ERROR
      at D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:218:35
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:28:103)
      at _next (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:30:194)
      at D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:30:364
      at new Promise (<anonymous>)
      at D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:30:97
      at _getUserByProviderAccountId (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:221:44)
      at getUserByProviderAccountId (D:\auction\work\social-link\node_modules\next-auth\dist\adapters\typeorm\index.js:195:44)
      at D:\auction\work\social-link\node_modules\next-auth\dist\server\lib\callback-handler.js:120:45
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (D:\auction\work\social-link\node_modules\next-auth\dist\server\lib\callback-handler.js:20:103)
      at _next (D:\auction\work\social-link\node_modules\next-auth\dist\server\lib\callback-handler.js:22:194)
]
https://next-auth.js.org/errors#oauth_callback_handler_error

```

**Comment by iaincollins** [comment]
:-(

We have made some progress on this issue in https://github.com/iaincollins/next-auth/issues/314 - probably best to track there! I have it working, but not really well yet.

tl;dr TypeORM handles these URLs a bit weirdly, but passing a database as an object (rather than URL) is a bit helpful.

**Comment by prettydev** [comment]
Thank you for your kind response.
Now I'm using Mongoose. Do I have to use TypeORM? 
And I hope you let me know the sample format for the .env to setting Mongo Atlas as an object.
